Assets/Scripts/BlockBehaviour.cs

```diff
@@ -264,11 +267,25 @@ public GameObject GetCollidableObject(BlockFace face)
 
     public void PlayDisplacementSound()
     {
-        AudioSource displacementSound = GetComponent<AudioSource>();
+        foreach (AudioSource sound in _soundList)
+        {
+            if (sound.clip.name == "BlockShifting" || sound.clip.name == "DisintegrateCharge")
+            {

```

Comment:
Shouldn't we have 1 displacement sound per block. And then we just call it. We don't need to have a sound list. There should be a cleaner way that allows us to add more shifting sounds.